client/watch.js

```diff
@@ -375,10 +388,17 @@ Template.watch_page.onRendered(function(){
   this.mainPlayerYTApiActivated = false;
   this.mainPlayerUSApiActivated = false;
   
+  this.checkTime = Meteor.setInterval(()=>{
+    if(mainPlayer && mainPlayer.getElapsedTime){
+      Session.set('currentTimeElapsed', mainPlayer.getElapsedTime());
+    }
+  }.bind(this),4000);
```

Comment:
I think I left .bind(this)  in from a prior attempt to pass the context, the fat rocket didnt seem to be doing the job. I've removed it now